File: edu/src/main/java/io/agora/edu/common/service/BoardService.java
package io.agora.edu.common.service;

import java.util.List;

import io.agora.edu.common.bean.board.BoardBean;
import io.agora.edu.common.bean.ResponseBody;
import io.agora.edu.common.bean.board.sceneppt.BoardCoursewareRes;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Path;

public interface BoardService {

    @GET("edu/apps/{appId}/v1/rooms/{roomUuid}/users/{userUuid}/resources")
    Call<ResponseBody<List<BoardCoursewareRes>>> getCourseware(
            @Path("appId") String appId,
            @Path("roomUuid") String roomUuid,
            @Path("userUuid") String userUuid
    );
}

File: edu/src/main/java/io/agora/edu/core/internal/server/requests/http/retrofit/services/deprecated/BoardService.kt
package io.agora.edu.core.internal.server.requests.http.retrofit.services.deprecated

import io.agora.edu.core.internal.edu.common.bean.ResponseBody
import io.agora.edu.core.internal.edu.common.bean.board.sceneppt.BoardCoursewareRes
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface BoardService {
    @GET("edu/apps/{appId}/v1/rooms/{roomUuid}/users/{userUuid}/resources")
    fun getCourseware(
            @Path("appId") appId: String?,
            @Path("roomUuid") roomUuid: String?,
            @Path("userUuid") userUuid: String?
    ): Call<ResponseBody<List<BoardCoursewareRes?>?>?>?
}